src/components/interaction-designer/BuilderCanvas.vue

```diff
@@ -165,7 +165,9 @@ export default class BuilderCanvas extends Vue {
     }
 
     const yPosition: number = this.blockAtTheLowestPosition.ui_metadata.canvas_coordinates.y
-    const scrollHeight = yPosition + this.blockHeight + MARGIN_HEIGHT
+    // TODO: clean this original solution in case the actual solution still works after we update the toolbar. I suspect there is a relation between the scrollHeight formula and the fact that we have toolbar elements with fixed position
```

Comment:
I added a todo in https://viamoinc.atlassian.net/browse/VMO-4192 to remind us about this

Reply:
I think we want to use the convention of either erasing the todo or putting the JIRA number right inside the code's todo comment so other people can refer to it and add details if they need to! Would you be willing to add the JIRA number into the code comment? Thanks!